src/processors/feature.ts

```diff
@@ -129,6 +152,23 @@ export class FeatureFileProcessor extends TypeScriptFileProcessor {
         return;
       }
 
+      if (FeatureFileProcessor.isExportAssignment(node)) {
+        const { expression } = node;
+        if (FeatureFileProcessor.isObjectLiteralExpression(expression)) {
+          const property = expression.properties[0];
+          if (FeatureFileProcessor.isPropertyAssignment(property)) {
+            const { name, initializer } = property;
+            if (
+              FeatureFileProcessor.isIdentifier(name) &&
+              FeatureFileProcessor.isStringLiteral(initializer) &&
+              name.escapedText === CSF3_EXPORT_TITLE_FIELD
+            ) {
+              callback(initializer.text);
```

Comment:
```suggestion
              callback(initializer.text);
              return;
```

can and should exit at this point as we've found the storybook name.